cardano-client/cardano-client.cabal

```diff
@@ -0,0 +1,29 @@
+name:                cardano-client
+version:             0.1.0.0
+synopsis:            An API for ouroboros-network
+-- description:
+license:             Apache-2.0
+license-files:       LICENSE
+copyright:           2019 Input Output (Hong Kong) Ltd.
+maintainer:
```

Comment:
I think it's good to put:
```
maintainer:          operations@iohk.io
```
For authors  it might be good to put `IOHK Engineering Team`.